addons/mail/static/src/components/discuss/tests/discuss_moderation_tests.js

```diff
@@ -1227,6 +1227,130 @@ QUnit.test('new pending moderation message posted by someone else when moderator
     );
 });
 
+QUnit.test('accept multiple moderation messages', async function (assert) {
+    assert.expect(9);
+
+    Object.assign(this.data.initMessaging, {
+        channel_slots: {
+            channel_channel: [{
+                id: 20,
+                is_pinned: true,
+                channel_type: "channel",
+                name: "general",
+                moderation: true,
+            }],
+        },
+        is_moderator: true,
+        moderation_counter: 1,
+        moderation_channel_ids: [20],
+    });
+    this.data['mail.message'].records = [{
+        author_id: [2, "Someone"],
+        body: "<p>test</p>",
+        id: 100,
+        model: 'mail.channel',
+        moderation_status: 'pending_moderation',
+        need_moderation: true,
+        res_id: 20,
+    }, {
+        author_id: [2, "Someone"],
+        body: "<p>test 2</p>",
+        id: 101,
+        model: 'mail.channel',
+        moderation_status: 'pending_moderation',
+        need_moderation: true,
+        res_id: 20,
+    }, {
+        author_id: [2, "Someone"],
+        body: "<p>test 3</p>",
+        id: 102,
+        model: 'mail.channel',
+        moderation_status: 'pending_moderation',
+        need_moderation: true,
+        res_id: 20,
+    }, {
+        author_id: [2, "Someone"],
+        body: "<p>test 4</p>",
+        id: 103,
+        model: 'mail.channel',
+        moderation_status: 'pending_moderation',
+        need_moderation: true,
+        res_id: 20,
+    }];
+
+    await this.start({
+        discuss: {
+            params: {
+                default_active_id: 'mail.box_moderation',
+            },
+        },
+    });
+
+    assert.containsN(
+        document.body,
+        '.o_Message',
+        4,
+        "should display 4 messages"
+    );
+
+    await afterNextRender(() => {
+        document.querySelectorAll('.o_Message_checkbox')[0].click();
+        document.querySelectorAll('.o_Message_checkbox')[1].click();
+    });
+    assert.containsN(
+        document.body,
+        '.o_Message_checkbox:checked',
+        2,
+        "2 messages should have been checked"
+    );
+    assert.doesNotHaveClass(
+        document.querySelector('.o_widget_Discuss_controlPanelButtonModeration.o-accept'),
+        'o_hidden',
+        "Global accept button should be displayed as two messages are selected"
```

Comment:
Strange mix of texts starting with lowercase and uppercase, I think we usually just lowercase everything in those. (double check this in all currently open PRs because I've seen it elsewhere too).